src/ledger/parse/ledger.js

```diff
@@ -8,6 +8,11 @@ import type {GetLedger} from '../types.js';
 function parseTransactionWrapper(ledgerVersion, tx) {
   const transaction = _.assign({}, _.omit(tx, 'metaData'),
     {meta: tx.metaData});
+
+  // normalize format from rippled
+  transaction.meta.delivered_amount = transaction.meta.DeliveredAmount;
```

Comment:
This is not correct if it's a partial payment.

> The Amount of a Payment transaction indicates the amount to deliver to the Destination, so if the transaction was successful, then the destination received that much -- except if the transaction was a partial payment. (In that case, any positive amount up to Amount might have arrived.) Rather than choosing whether or not to trust the Amount field, you should use the delivered_amount field of the metadata to see how much actually reached its destination.

https://ripple.com/build/transactions/#delivered-amount